winui/Rating/Precision.md

```diff
@@ -13,7 +13,7 @@ The rating value can be provided with three options such as full, half, and exac
 
 ## Full
 
-Rating value can be provided with whole precision.
+Rating value can be provided with nearest rounded value.
```

Comment:
Rating value can be provided with the nearest rounded value.

Reply:
Changed the content as mentioned above.